app/views/dashboard.pug

```diff
@@ -20,19 +20,19 @@ html(lang="en")
                 h1 Welcome to Hive!
 
         // Side Button Bar
-        div(id='navBar' style='position:absolute;width:10%;height:100%;left:0%;top:0%;border-style:solid;background-color:rgb(49,60,80);')
+        nav(class="navbar navbar-nav" id='navBar' style='position:absolute;width:10%;height:100%;left:0%;top:0%;border-style:solid;background-color:rgb(49,60,80);')

```

Comment:
if so, yes. I'm pretty sure it let's bootstrap know it's not a default navbar, that way it doesn't try to force a horizontal bar.